### ISSUE TITLE
"Null" text icon in EDGE

### ISSUE THREAD

**Issue by endroca** [akveo/nebular#1962]
### Issue type

* [x] bug report
* [ ] feature request

### Issue description

When accessing <nb-icon> with edge browser appears "null" for third party packs

![401b735e-cc02-454c-bdda-667bf406fc01](https://user-images.githubusercontent.com/6523466/65094668-4bb43800-d994-11e9-8514-6e59f31df160.jpeg)

http://akveo.com/ngx-admin/pages/ui-features/icons

### Other information:

**Browser**
```
Edge: 44.18362.329.0
EdigeHTML: 18.18362
```

**Angular, Nebular**
```
Nebular: 4.1.3
```

**Comment by atr2600** [comment]
What I am seeing is edge is adding null into the <nb-icon>null</nb-icon> element. 

```
<nb-icon class="menu-icon ng-tns-c7-7 fas fa-brain ng-star-inserted" _nghost-voh-c6="" ng-reflect-config="brain">null</nb-icon>
```

Maybe we can do some kind of hacky fix to remove nulls before the page is loaded? 

But that does not answer why it is doing that.

**Comment by atr2600** [comment]
```
import { Component, OnInit } from '@angular/core';
// HERE IS THE FIX
import * as $ from 'jquery';

@Component({
  selector: 'app-home',
  templateUrl: './home.component.html',
  styleUrls: ['./home.component.css']
})
export class HomeComponent implements OnInit {

  constructor() { }

  ngOnInit() {
    // tslint:disable-next-line:triple-equals
    $('nb-icon').contents().filter(function() { return this.nodeType != 1; }).remove();
  }

}
```

if you add import ```import * as $ from 'jquery';```
and add
```
    // tslint:disable-next-line:triple-equals
    $('nb-icon').contents().filter(function() { return this.nodeType != 1; }).remove();
```
inside ngOnInit() then it solves the problem.

**Comment by yetanotherspinner** [comment]
It seems that adding "else" branch to icon.component.ts fixes it (at least for IE):
```
    if (content) {
      this.html = this.sanitizer.bypassSecurityTrustHtml(content);
    }
    else { this.html = ""; }
```

**Comment by yggg** [comment]
Fixed by #2043

**Comment by YassinMohamadi** [comment]
For internet explorer 11, you can change your CSS like that :
`nb-icon.menu-icon{
  visibility: hidden;
}
nb-icon.menu-icon::before{
  visibility: visible;
}`
